shardingsphere-control-panel/shardingsphere-cluster/shardingsphere-cluster-heartbeat/src/main/org/apache/shardingsphere/cluster/heartbeat/task/HeartBeatTaskManager.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.cluster.heartbeat.task;
+
+import com.google.common.base.Preconditions;
+
+import java.util.Timer;
+
+/**
+ * Heart beat task manager.
+ */
+public final class HeartBeatTaskManager {
+    
+    private static final String TIMER_NAME = "ShardingSphere-Cluster-HeartBeat";
+    
+    private Integer interval;
+    
+    private final Timer timer;
```

Comment:
Is `Executors.newSingleThreadScheduledExecutor()`  a good way?

Reply:
Yes, threadpool is better, i will fix it.